packages/roosterjs-editor-plugins/lib/plugins/Paste/sanitizeHtmlColorsFromPastedContent/sanitizeHtmlColorsFromPastedContent.ts

```diff
@@ -0,0 +1,31 @@
+import { chainSanitizerCallback, getTagOfNode } from 'roosterjs-editor-dom';
+import { DeprecatedColorList } from './deprecatedColorList';
+import { HtmlSanitizerOptions } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ * Remove the deprecated colors from pasted content
+ * @param fragment the pasted fragment
+ * @param sanitizingOption the sanitizingOption of BeforePasteEvent
+ * */
+export default function sanitizeHtmlColorsFromPastedContent(
+    fragment: DocumentFragment,
+    sanitizingOption: Required<HtmlSanitizerOptions>
+) {
+    const htmlElements = fragment.querySelectorAll('*') as NodeListOf<HTMLElement>;
+    const allTags = Array.from(htmlElements).map(el => getTagOfNode(el));
+    const uniqueTags = allTags.filter((tag, index) => allTags.indexOf(tag) == index);
+
+    uniqueTags.forEach(tag =>
+        chainSanitizerCallback(sanitizingOption.elementCallbacks, tag, element => {
```

Comment:
Instead of using elementCallbacks, you can use cssStyleCallbacksm to specify just color and background color, so that no need to query all unique tag names.